worker/test/src/RTC/Codecs/TestVP8.cpp

```diff
@@ -387,3 +448,88 @@ SCENARIO("process VP8 payload descriptor", "[codecs][vp8]")
 		REQUIRE(context.GetCurrentTemporalLayer() == 0);
 	}
 }
+
+SCENARIO("encode VP8 payload descriptor", "[codecs][vp8]")
+{
+	/** VP8 Payload Descriptor
+	 *
+	 * 1 = X bit: Extended control bits present (I L T K)
+	 * 0 = R bit: Reserved for future use
+	 * 0 = N bit: Reference frame
+	 * 0 = S bit: Continuation of VP8 partition
+	 * 000 = Part Id: 0
+	 * 1 = I bit: Picture byte ID
+	 * 1 = L bit: TL0PICIDX byte present
+	 * 1 = T bit: TID (temporal layer index) byte present
+	 * 0 = K bit: TID/KEYIDX byte present
+	 * 0000 = Reserved A: 14
+	 * 0000000000000001 = PictureId
+	 */
+
+	// clang-format off
+	uint8_t buffer[] =
+	{
+		0x80, 0xe0, 0x01, 0x01,
+		0xe8, 0x40, 0x7a, 0xd8
+	};
+
+	// clang-format on
```

Comment:
Double comment here?

Reply:
And/or wrong blank line before the clang-format comment.

Reply:
@jmillan ☝🏻